templates/support.html

```diff
@@ -30,6 +31,30 @@ <h2>Join the Matrix.org Foundation</h2>
         </p>
         <a href="/membership" class="call-to-action">Become a member</a>
     </div>
+    <div id="supporters">
+        <div class="supporters-section">
```

Comment:
```suggestion
        <div id="gold-supporters" class="supporters-section">
```

Reply:
Applied in c3118df625fdeb83ca1abdbbc26db8293f376db4